examples/server/app/movies/main.py

```diff
@@ -16,7 +17,7 @@
 from bokeh.sampledata.movies_data import movie_path
 
 conn = sql.connect(movie_path)
-query = open(join(dirname(__file__), 'query.sql')).read()
+query = (Path(__file__).parent / 'query.sql').read_text(encoding="utf8")
```

Comment:
If `encoding` is not kwarg-only (I don't think it is) I would prefer this throughout:
```suggestion
query = (Path(__file__).parent / 'query.sql').read_text("utf8")
```

Reply:
Hi @graingert, are you still working on this issue? Or would you like someone else to finish it?

Reply:
I'm happy to let someone else finish this

Reply:
OK. I tried pushing to your repo but I'm getting this error:

`git push https://github.com/graingert/bokeh.git 12779:enable-filterwarnings-error`

`remote: Permission to graingert/bokeh.git denied to Azaya89.`

`fatal: unable to access 'https://github.com/graingert/bokeh.git/': The requested URL returned error: 403`

`12779` is the name of my local branch created for this PR. I don't know if I made a mistake here...

Reply:
@Azaya89 I think you need to be a member of bokeh to be able to push changes to contributor PR branches

Reply:
Ooh. Alright then. It appears you've already updated the changes too :)

Reply:
@Azaya89 I can add you to @bokeh/dev if you like